src/instruction.rs

```diff
@@ -1,5 +1,6 @@
 /// MIPI DSI compatible instructions.
 #[allow(dead_code)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
 #[repr(u8)]
 pub enum Instruction {
```

Comment:
This enum should be split into two parts: All codes from 0x00 to 0xAF are common for all MIPI DCS devices and the codes >= 0xB0 are the manufacturer command set, which is device dependent. While some manufacturer commands are common among devices there are also differences: 0xB1 is, for example, a different command on a ILI9342 and a ST7789.

Reply:
I think I will remove this enum completely and just use the DcsCommand structures with direct hex values. It serves no useful purpose to have those values in two places.

Reply:
SGTM